spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -701,6 +709,11 @@
                                           completed_at: 2.weeks.from_now,
                                           order_cycle: oc3 )
         }
+        let!(:o5) {
+          create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                          completed_at: 100.days.ago,
+                                          order_cycle: oc4 )
+        }
```

Comment:
Can we have an expectation for this new record?

Reply:
I checked out the code and I think that it's worth checking for the line items too. 
I couldn't help but try to add it, and sorry then I did some refactoring along the way.... (that was kinda fun 😄 )